webapp/src/js/controllers/inbox.js

```diff
@@ -52,6 +53,20 @@ var feedback = require('../modules/feedback'),
   ) {
     'ngInject';
 
+    window.startupTimes.angularBootstrapped = performance.now();
+    Telemetry.record(
+      'boot_time_1_to_first_code_execution',
```

Comment:
So the names here are basically `namespace_num_description`. The number is just so you know ordering of each section. There is a namespace, but the namespace doesn't use a period because I thought it would be weird to have a period in a JSON property name.

The current aggregation is super duper simple, it just works out `_stats` for each key, and so keys can't, at least as the core identifier, be namespaced specifically at that point. Any namespacing would just be convention.

It wouldn't be hard though, to provide a little lift to the convention and change the aggregation so that `boot_time.1_to_first_code_execution`, once aggregated, ended up being:

```json
{
	"boot_time": {
		"1_to_first_code_execution": "blah",
		"2_to_bootstrap": "blah"
	}
}
```

Is that what you're interested in here?

Reply:
I'm not fussed if it's implicit or explicit, either in the data model or Telemetry.record()` calls, but I think using a different divider for the namespace would:
* make it more obvious what the namespace was, e.g. `namespace.metric`, or `namespace:metric` if we're uncomfortable having `.` in a JSON property name
* make it possible to filter metrics by namespace when using simple tools like `grep`, `cut` etc.